code/src/components/App/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import {
+  createMuiTheme,
+  ThemeProvider,
+  responsiveFontSizes
+} from '@material-ui/core/styles';
+import { CssBaseline } from '@material-ui/core';
+import store from 'store';
+import Pages from 'pages';
```

Comment:
remember to separate imports from node-modules and from your src folder with an empty line 🤓